Application/EdFi.Ods.Api/Container/Modules/ExtensionsModule.cs

```diff
@@ -64,27 +64,22 @@ public override void ApplyConfigurationSpecificRegistrations(ContainerBuilder bu
 
                     var relationshipContextDataProviderTypes = assembly.GetTypes()
                         .Where(
-                            t => !t.IsAbstract && typeof(IRelationshipsAuthorizationContextDataProvider<,>).IsAssignableFromGeneric(t))
+                            t => !t.IsAbstract && typeof(IRelationshipsAuthorizationContextDataProvider<>).IsAssignableFromGeneric(t))
```

Comment:
Removed the unnecessary complexity of an old extensibility point (the generic type `TContextData` which was constrained to derive from `RelationshipsAuthorizationContextData`). This mechanism has never actually been used and was causing unnecessary complexity with the container registrations impacted by this ticket.